FRENCH/src/translation-terms.md

```diff
@@ -54,6 +56,7 @@ français.
 | floating-point number | nombre à virgule flottante | - |
 | function | fonction | - |
 | generics | génériques / généricité | - |
+| generic type parameter | paramètre de type générique | - |

```

Comment:
Not used in this chapter, shouldn't be part of this merge

Reply:
Yes, I remember I have done this mistake only one time, a long time ago, it's probably here.
I don't think it is very important, so please do several separated Push Request on concerned chapters if you want to do this work :wink: